packages/amazon-sumerian-hosts-babylon/src/Babylon.js/HostObject.js

```diff
@@ -141,33 +141,80 @@ class HostObject extends CoreHostObject {
     scene,
     {
       modelUrl,
-      animStandIdleUrl,
-      animLipSyncUrl,
-      animGestureUrl,
-      animEmoteUrl,
-      animFaceIdleUrl,
-      animBlinkUrl,
-      animPointOfInterestUrl,
+      animUrls,
       gestureConfigUrl,
       pointOfInterestConfigUrl,
     },
   ) {
-    // Load character model
-    const characterAsset = await BABYLON.SceneLoader.LoadAssetContainerAsync(modelUrl, undefined, scene);
+    const characterAsset = await this.loadCharacterMesh(scene, modelUrl);
     const characterMesh = characterAsset.meshes[0];
+    const bindPoseOffset = characterAsset.animationGroups[0];
 
-    // rename mesh to something human-readable instead of the default '__root__'
-    characterMesh.name = modelUrl.slice(modelUrl.lastIndexOf('/') + 1, modelUrl.lastIndexOf('.'));
```

Comment:
Note that I removed this functionality from HostObject altogether - I decided that only the BabylonJS editor cared about human-readable names here, so I moved it to the plugin